config/environments/production.rb

```diff
@@ -90,8 +86,6 @@
     host: Figaro.env.domain_name,
     protocol: 'https'
   }
-  config.action_mailer.delivery_method = Figaro.env.pt_mode == 'on' ? :letter_opener_web : :smtp
-  config.action_mailer.perform_deliveries = true
```

Comment:
Yes, since its redundant since true its the default.    I could re-add it commented out if prefer?